src/lib.rs

```diff
@@ -142,7 +142,7 @@ pub trait BufRead {
     ///
     /// No guarantees are provided about the contents of `buf` when this function is called, so
     /// implementations cannot rely on any property of the contents of `buf` being true. It is
-    /// recommended that implementations only write data to `buf` instead of reading its contents.
+    /// recommended that implementations only write data to `buf` and do not read its contents.
```

Comment:
Heh, since this is [copy-pasted from `std`](https://doc.rust-lang.org/stable/std/io/trait.Read.html#method.read_exact) you might want to make a PR there too. :)

Reply:
Ah this was a very minor change among the other grammar changes man, the original is technically correct. I won't bother PR'ing this change on its own to std.